From a81ff4bd9c0d917d10be0de4dd8d8e20b449b531 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 10 May 2017 15:37:28 +0000
Subject: [PATCH 0764/1000] MANTIS 40176 extension: Components for
 enabling/disabling TouchScreen and Display

textidl.lib
V1.0.0.14

- added commandType CMD_SWITCH_DISPLAY


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1198 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp   | 9 +++++++--
 .../Massai/cpp/textIDL/src/textappcontrol.cpp            | 3 ++-
 2 files changed, 9 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
index 0023047d..3e11e140 100644
--- a/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
@@ -1,6 +1,6 @@
 #include "tools/versionlib.h"
 
-VERSION_ID_DT(textIDL, 1, 0, 0, 13);
+VERSION_ID_DT(textIDL, 1, 0, 0, 14);
 
 /*!
 \file
@@ -8,12 +8,17 @@ VERSION_ID_DT(textIDL, 1, 0, 0, 13);
 
 \page textidl.lib Release Notes - textidl.lib
 
-\version 1.0.0.13
+\version 1.0.0.14
 
 \section changehistory Change history
 
 textidl.lib
 
+\version 1.0.0.14
+\li 05.05.2017
+\li SteT Mantis 40176: extension: Components for enabling/disabling TouchScreen and Display
+- added commandType CMD_SWITCH_DISPLAY
+
 \version 1.0.0.13
 \li 18.01.2017
 \li Mantis 0044241: EZAccess8KeyAudio
diff --git a/SelfServiceCommon/Massai/cpp/textIDL/src/textappcontrol.cpp b/SelfServiceCommon/Massai/cpp/textIDL/src/textappcontrol.cpp
index d3fe31e0..ee928f7f 100644
--- a/SelfServiceCommon/Massai/cpp/textIDL/src/textappcontrol.cpp
+++ b/SelfServiceCommon/Massai/cpp/textIDL/src/textappcontrol.cpp
@@ -32,7 +32,7 @@ namespace MASSAI
         default:                        return unknown("displayState", state);
       }
     }
-    
+
     std::string commandTypeText(const appControl::commandType& code)
     {
       switch(code)
@@ -44,6 +44,7 @@ namespace MASSAI
         case appControl::CMD_SWITCHMAIN:     return "switchmain";
         case appControl::CMD_SHOW:           return "show";
         case appControl::CMD_SET_RESOLUTION: return "set_resolution";
+        case appControl::CMD_SWITCH_DISPLAY: return "switch display";
         default:                             return unknown("commandType", code);
       }
     }
-- 
2.41.0.windows.1

